src/components/editor/editor-pane/tool-bar/tool-bar.scss

```diff
@@ -2,8 +2,10 @@
   border: 1px solid #ededed;
 }
 
-.divider {
+.btn-toolbar .btn-group:not(:last-of-type)::after {
```

Comment:
Please move this code block into the rule above as sub rule.

Reply:
Done